src/init.cpp

```diff
@@ -956,6 +957,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     fIsBareMultisigStd = GetBoolArg("-permitbaremultisig", true);
     nMaxDatacarrierBytes = GetArg("-datacarriersize", nMaxDatacarrierBytes);
+    nRbfPolicy = GetArg("-replacebyfee", nRbfPolicy, 1);
```

Comment:
I think the logic was that this would be the most likely rbf policy people would want to use other than the default. I would be ok with changing it to be more explicit.